verl/utils/fs.py

```diff
@@ -290,3 +299,78 @@ def local_mkdir_safe(path):
         os.makedirs(path, exist_ok=True)
 
     return path
+
+
```

Comment:
please either make copy_local_path_from_remote private (`_copy_local_path_from_remote`) or move the implementation directly to `copy_to_local`.